xwiki-platform-core/xwiki-platform-dashboard/xwiki-platform-dashboard-ui/src/main/resources/Dashboard/XWikiUserDashboardSheet.xml

```diff
@@ -175,4 +175,147 @@
+        <prettyName>Asynchronous rendering</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </async_enabled>
+      <content>
+        <disabled>0</disabled>
+        <editor>Text</editor>
+        <name>content</name>
+        <number>1</number>
+        <prettyName>Executed Content</prettyName>
+        <rows>25</rows>
+        <size>120</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </content>
+      <extensionPointId>
+        <disabled>0</disabled>
+        <name>extensionPointId</name>
+        <number>5</number>
+        <prettyName>Extension Point ID</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </extensionPointId>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>6</number>
+        <prettyName>Extension ID</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parameters>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>parameters</name>
+        <number>7</number>
+        <prettyName>Extension Parameters</prettyName>
+        <rows>10</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </parameters>
+      <scope>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>scope</name>
+        <number>8</number>
+        <prettyName>Extension Scope</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>wiki=Current Wiki|user=Current User|global=Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </scope>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <content/>
+    </property>
+    <property>
+      <extensionPointId>org.xwiki.plaftorm.user.profile.menu.content</extensionPointId>
+    </property>
+    <property>
+      <name>org.xwiki.platform.user.profile.entry.userdashboardsheet</name>
+    </property>
+    <property>
+      <!-- display: The user is seeing her own profile and the dashboard document exists -->
+      <parameters>id=dashboard
```

Comment:
Is this id needed given than UI extensions already have an id ?

Reply:
The id is used in the url (e.g., http://localhost:8080/xwiki/bin/view/XWiki/Admin?category=notifications, here the id is notifications). I think changing this would break any bookmarked link to the pages of the menu.

Reply:
I can understand the retro compatibility need but it should at least not be mandatory and fallback on uix extension id when not provided IMO. WDYT ?

Reply:
Sounds good. I'll do this.

Reply:
> The id is used in the url (e.g., http://localhost:8080/xwiki/bin/view/XWiki/Admin?category=notifications, here the id is notifications). I think changing this would break any bookmarked link to the pages of the menu.

I'm not sure about this. I don't consider this to be a public API.  AFAIK it's also not documented anywhere.

Reply:
> I don't consider this to be a public API

It's not something exotic for an extension to lead to a specific well know user profile section.

Reply:
Preserving the id while making it optional for new UIX is quite easy (3 LoC) and would preserve existing bookmark to user menu pages.
Additionaly, I personally consider this kind of links are part of a public API and would argue to keep them.